src/js/Rickshaw.Graph.Band.js

```diff
@@ -12,9 +12,18 @@ Rickshaw.Graph.Band = Rickshaw.Class.create({
     var bands = args.bands || graph.bands
 
     var vis = args.vis || graph.vis
```

Comment:
```
var graph = this.graph
var bands = args.bands || graph.bands
var vis = args.vis || graph.vis

var config = graph.renderer.config
var configRenderer = !!config && config.renderer

// add comment why 'multi' required for `selectAll`
var selection = !!configRenderer && configRenderer === 'multi' ?
  vis.selectAll('*') :
  vis
```

Reply:
while what you wrote works, perhaps take advantage of a ternary and add a comment as to why 'multi' is required - it would give more context here and less required eyeballing for variables are reassigned based on if-statements.